docs/discussion_note_boringen_methods.rst

```diff
@@ -0,0 +1,115 @@
+           """
+           pass
+   
+   
+   class DovBoringen(DovSearch):
+       def __init__(self, location):
+           """instantiate class for certain location
+           
+           location can be anything from coordinates (with buffer), bbox
+           or polygon
+           """
+           self.location = location # add method to derive location from input
+           pass
+
+       def list_interpretations(self, ):
+           """check which intepretations are available
+           """
+           pass
+
+       def get_interpretation(self, interpretation):
+           """get data from wfs and/or xml for a certain interpretation
+           
+           Parameters
+           ----------
+           interpretation: string
+                the selected intepretation
+           
+           """
+           self.ip = globals()[interpretation]()
+           df_boring .... get data from....
+           data_ip ... get data from....
+           df_ip = self.ip.get_interpretation(data)
+           # add method to join with where clause
+           return df
+
+
+   class Interpretatie(DovSearch):
+       """class for interpretation related stuff
+       """
+       def __init__(self,):
+           """instantiate class 
+           """
+           self.defined_interpretations = ['InformeleStratigrafie',
+                                           'FormeleStratigrafie',
+                                           'Lithologische beschrijvingen',
+                                           'GecodeerdeLithologie',
+                                           'HydrogeologischeStratigrafie',
+                                           'InformeleHydrogeologischeStratigrafie',
+                                           'QuartaireStratigrafie',
+                                           'GeotechnischeCoderingen',]
+
+
+   class HydrogeologischeStratigrafie(Interpretatie):
+       """class for interpretation of Hydrogeological Stratification
+       """
+       def __init__(self, location=None):
+           """instantiate class for certain location
+           
+           location can be anything from coordinates (with buffer), bbox
+           or polygon, default None
+           """
+           if location:
+               self.location = location # add method to derive location from input
+           self.headers = ['pkey_interpretatie', 
+                           'pkey_boring', 
+                           'pkey_sondering', 
+                           'diepte_laag_van',
+                           'diepte_laag_tot',
+                           'aquifer']
+           
+       def get_dataframe(self, input):
+           """create dataframe from input
+           
+           """
+           self.df = pd.DataFrame(input, columns=self.headers) 
+
+   """
+   Examples
+    --------
+   >>> boring = DovBoringen(location)
+   >>> boring_metadata = boring.get_metadata()
+   >>> boring_data = boring.get_data()
```

Comment:
With respect to the `DovGrondwaterFilter` this would impy:
default the user get the `Ligging` (purely WFS) as dataframe and can additionaly ask for the `Observaties` or the `Peilmetingen` to join with these...